paper/paper.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Retriever: Data Retrieval Tool'
+tags:
+  - data retrieval, dataset, python, data, data science, datasets
+authors:
+ - name: Henry Kironde
+   orcid: 0000-0001-7105-5808
+   affiliation: 1
+ - name: Benjamin D. Morris
+   orcid: 0000-0003-4418-1360
+ - name: Ethan P. White
```

Comment:
I should be moved to last place in the author list since the convention in biology is for the head of the main lab to be in the last position.